subprojects/docs/src/docs/userguide/developing-plugins/implementing_gradle_plugins.adoc

```diff
@@ -256,6 +256,18 @@ The same concept applies to task types.
 include::{snippetsPath}/developingPlugins/reactingToPlugins/groovy/buildSrc/src/main/java/InhouseConventionWarPlugin.java[]
 ----
 
+[[sec:depending_on_plugins]]
+== Depending on plugins
+
+A plugin sometimes requires the use of other plugins.
+Depending on where those other plugins come from you might need to declare dependencies.
+
+When your plugin reacts to the application of a core Gradle plugin or of a plugin whose implementation resides in the same Gradle project, you don't need to declare any dependency because it respectively comes from the Gradle distribution or is directly available.
```

Comment:
💭 
I am not sure of the word reacts here.

Reply:
See the section right above